Sources_v3/StreamChatUI/UIConfig.swift

```diff
@@ -41,6 +41,7 @@ public extension UIConfig {
     struct Navigation {
         public var navigationBar: ChatNavigationBar.Type = ChatNavigationBar.self
         public var channelListRouter: ChatChannelListRouter<ExtraData>.Type = ChatChannelListRouter<ExtraData>.self
+        public var messageListRouter: ChatChannelRouter<ExtraData>.Type = ChatChannelRouter<ExtraData>.self
```

Comment:
nit: we don't really use "message list" as a term for chat detail. Maybe `channelDetailRouter`?